src/app/sidebar/components/Card.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Heading } from "@chakra-ui/react";
+import React from "react";
+
+export interface CardProps {
+  title: string;
+  subject: string;
+  code: string;
+  selected?: boolean;
+}
+
+export function Card(props: CardProps): JSX.Element {  
```

Comment:
@aomi I usually build components like this, so that I don't need to call props.SOMETHING every time. Do you have a preference?
```
export const Card: React.FC<CardProps> = ({
  title,
  subject,
  code,
  selected
}) => {
```

Reply:
I would personally go with 
```tsx
export const Card = ({title}: CardProps): JSX.Element => {};
```
if we wanted to use arrow functions but.

Reply:
I mainly told Sunwoo to write it this way as I found it the simplest to explain as functions are pretty straightforward.

Reply:
It seems like it's best practice to leave it the way it is: https://github.com/facebook/create-react-app/pull/8177